src/App.js

```diff
@@ -99,13 +109,17 @@ export default function App() {
   );
 }
 
+const UnderlinedHeading = styled.h1`
+  text-decoration: underline;
+`;
+
 const OrganizingMain = styled.main`
   display: flex;
   flex-direction: column;
   align-items: center;
   max-width: 500px;
   margin: 0 auto;
-  padding: 15px;
+  padding: 0 15px 3vh 15px;
```

Comment:
I would recommend you not to mix the units. It works, but on the whole it is more difficult understand, what's going on. But that's minor note.

Reply:
this was a lot of adjusting 1px forth and back in the first place, i changed this now to %